third_party/3/pyspark/sql/readwriter.pyi

```diff
@@ -54,7 +54,7 @@ class DataFrameWriter(OptionUtils):
     def sortBy(self, col: TupleOrListOfString) -> DataFrameWriter: ...
     def save(self, path: Optional[str] = ..., format: Optional[str] = ..., mode: Optional[str] = ..., partitionBy: Optional[List[str]] = ..., **options: Any) -> None: ...
     def insertInto(self, tableName: str, overwrite: Optional[bool] = ...) -> None: ...
-    def saveAsTable(self, name: str, format: Optional[str] = ..., mode: Optional[str] = ..., partitionBy: Optional[List[str]] = ..., **options: str) -> None: ...
+    def saveAsTable(self, name: str, format: Optional[str] = ..., mode: Optional[str] = ..., partitionBy: Optional[List[str]] = ..., **options: Any) -> None: ...
```

Comment:
In general I still haven't made my mind if I am fine with making this annotation less strict. But just for the sake of argument - `Optional[Union[bool, int, float str]]` (extended https://github.com/zero323/pyspark-stubs/issues/275#issue-544150559) is something that is up for discussion as:

- There is reasonable justification for it - there are already `option` methods with `Bool`, `Double`, `Long`, `String`.
- These are rather unlikely to be extended and / or modified by the final user, so string representation is predictable.
- Smaller set of types is usually preferred, as it helps catching some naive mistakes.
- In case of unlikely event that this project is  merged with upstream and inlined, we really want to avoid `Any` (stubs don't validate against the annotated source, so it doesn't matter much at the moment).